libs/designsystem/testing-base/src/lib/components/mock.dropdown.component.ts

```diff
@@ -1,6 +1,8 @@
-import { forwardRef, Component, Input, Output, EventEmitter } from '@angular/core';
-
-import { DropdownComponent } from '@kirbydesign/designsystem';
+import { Component, EventEmitter, forwardRef, Input, Output } from '@angular/core';
+import {
+  cardAlignment,
+  DropdownComponent,
+} from 'libs/designsystem/src/lib/components/dropdown/dropdown.component';

```

Comment:
Please import from lib namespace (add `cardAlignment` to `components/index.ts` barrel file if needed) :
```suggestion
} from '@kirbydesign/designsystem';

```

Reply:
Done